docs/source/nlp/text_normalization/wfst/wfst_customization.rst

```diff
@@ -3,9 +3,9 @@
 Grammar customization
 =====================
 
-.. note::
+.. warning::
 
-    TN/ITN is transitioning from `NVIDIA/NeMo <https://github.com/NVIDIA/NeMo>`_ repository to a standalone `NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_ repository. All updates and discussions/issues should go to the new repository.
+    TN/ITN transitioned from `NVIDIA/NeMo <https://github.com/NVIDIA/NeMo>`_ repository to a standalone `NVIDIA/NeMo-text-processing <https://github.com/NVIDIA/NeMo-text-processing>`_ repository. All updates and discussions/issues should go to the new repository.
```

Comment:
nit: are we planning to remove these sections in the NeMo doc, or cross-ref the NeMo text processing repo doc here? if it is the former, please specify which release we will remove it. For example, we could say "we will remove this section on the release 1.17.0. All updates..."

Reply:
the latter